junit-jupiter-api/src/main/java/org/junit/jupiter/api/IndicativeSentencesSeparator.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2015-2019 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * https://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junit.jupiter.api;
+
+import static org.apiguardian.api.API.Status.EXPERIMENTAL;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.apiguardian.api.API;
+
+/**
+ * {@code @IndicativeSentencesSeparator} is used to declare a custom separator
+ * in the IndicativeSentencesGenerator, if this notation was not declared
+ * IndicativeSentencesGenerator will use ", " as separator by default.
+ *
+ * @since 5.6
+ * @see DisplayNameGenerator
+ */
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+@Inherited
+@API(status = EXPERIMENTAL, since = "5.6")
+public @interface IndicativeSentencesSeparator {
```

Comment:
Let's rename this annotation to `@IndicativeSentencesGeneration`, rename `value` to `separator` (so we can easily add additional config options in the future), and meta-annotate it with `@DisplayNameGeneration(IndicativeSentences.class)`.

Reply:
Sure!, on it with the refactor